debian/kafkacat/Dockerfile

```diff
@@ -1,14 +1,33 @@
-FROM debian:stable
+FROM debian:stretch-slim
+
+WORKDIR /build
+
+ENV VERSION=1.3.1-1
+
+ENV BUILD_PACKAGES="build-essential cmake python git curl zlib1g-dev libsasl2-dev libssl-dev"
+
+RUN echo "Building kafkacat ....." \
```

Comment:
Because you are doing multistage, and therefore the layering won't matter, you could break this up to be more readable over multiple RUNs.

Reply:
side by side they're about as readable, so i'll leave it for now.